edge/conf/edge.yaml

```diff
@@ -46,6 +46,7 @@ edged:
     remote-image-endpoint: unix:///var/run/dockershim.sock
     runtime-request-timeout: 2
     podsandbox-image: k8s.gcr.io/pause
+    image-pull-progress-deadline: 60 # second
```

Comment:
does keadm need to be changed as well?

Reply:
I think not required. As the default value is set already.

Also we need to modify keadm handling the configurations from flags at it will be too many flags. Need to think of a simpler way as we cannot expose all configurations using flags.

Reply:
Yes agree, too many flags are easy to get confused